etc/kayobe/environments/ci-tenks/kolla/config/bifrost/bifrost.yml

```diff
@@ -0,0 +1,5 @@
+# yamllint disable-file
+---
+# Don't build an IPA deployment image, instead download upstream images.
+create_ipa_image: false
+download_ipa: true
```

Comment:
Do you need to set these? I thought Bifrost defaults to downloading

Reply:
Removed file